UnityProject/Assets/Scripts/Core/Input System/MouseInputController.cs

```diff
@@ -430,6 +432,20 @@ private bool CheckHandApply(GameObject target)
 		return false;
 	}
 
+	/// <summary>
+	/// Used if you want to Force an interaction, Between Local character and certain Script ( Skips handApply.HandObject  )
+	/// </summary>
+	/// <param name="RelatedApply"></param>
+	/// <param name="Target"></param>
+	public static void CheckHandApply(IBaseInteractable<HandApply> targetHandAppliable, GameObject Target)
+	{
+		//call the used object's handapply interaction methods if it has any, for each object we are applying to
+		var handApply = HandApply.ByLocalPlayer(Target);
+
+		if (targetHandAppliable.ClientCheckAndTrigger(handApply)) ;
+
```

Comment:
This If supposed to do something?

Reply:
yeah, if you wanted to manually trigger an interaction on a script but opted for net message instead

Reply:
but I *

Reply:
```suggestion
		targetHandAppliable.ClientCheckAndTrigger(handApply);
```
Yeah but its not doing anything so you can just put